src/connection.go

```diff
@@ -35,8 +42,7 @@ func (c configConnectionError) Error() string {
 	return "can't execute redis 'CONFIG' command: " + c.cause.Error()
 }
 
-func newRedisCon(hostname string, port int, unixSocket string, password string) (conn, error) {
-
+func newRedisCon(hostname string, port int, unixSocket string, password string, renamedCommands map[string]string) (conn, error) {
```

Comment:
I would remove `renamedCommands` from the constructor here, as the zero value behaves fine and is a reasonable default. We can set the map of the conn object later after building it.

Reply:
I have removed it and now is populating `renamedCommands map[string]string` afterwards